unskript-ctl/templates/last_cell_content.j2

```diff
@@ -7,6 +7,20 @@ try:
                 output = json.loads(output)
                 output['id'] = id
                 print(json.dumps(output))
+            # Lets take care of Timed out checks too. That output
+            # will be not included in the workflow output. Lets findout
+            # and explictly add them to the output
+            if len(w.check_output.keys()) < len(w.check_uuids):
```

Comment:
I think this check is inappropriate. We should check for what timed out. 
By checking like this, we are inadvertently covering up other issues where this condition can occur.

In the timeout handler, we should append the timed out check to a list or something and use that list here.

Reply:
I beg to differ here. The workflow object catches Two off the Three possible states. Two of them, `Pass` and `Fail` are recorded in the Workflow object. The Errored / Timeout case was missing, meaning the try/catch in the `_run_function` routine basically logs the error and continues with the execution. This is the missing part.  The idea here is check_uuids has every check that needs to be run .the check_output.keys() is the final result of Pass and Fail cases.  So the difference is the Errored case, hence the approach that i took seems to right here. 

I think this implementation is correct.